inc/command/cleanticketscommand.class.php

```diff
@@ -43,7 +43,7 @@ class CleanTicketsCommand extends Command
 {
    protected function configure() {
       $this
-         ->setName('glpi:plugins:formcreator:clean_tickets')
+         ->setName('plugins:formcreator:clean_tickets')
```

Comment:
Maybe the old name should be kept as an alias and only be removed on the next major version ? 
Same for the others commands names changes.